docs/src/tutorial/cython_tutorial.rst

```diff
@@ -338,8 +418,27 @@ standard library.
     how many elements you are going to put in the vector. For more details
     see `this page from cppreference <https://en.cppreference.com/w/cpp/container/vector>`_.
 
-.. literalinclude:: ../../examples/tutorial/cython_tutorial/primes_cpp.pyx
-    :linenos:
+.. tabs::
+    .. group-tab:: Pure Python
+
+        .. literalinclude:: ../../examples/tutorial/cython_tutorial/primes_cpp.py
+            :linenos:
+
+        .. warning::
+
+            This code uses ``cimports``, cython understands this
+            and imports the necessary modules, but python cannot do this,
+            therefore it will not work and will raise an exception.
+
+            And since there is no direct analogue of a vector in python,
+            with the same functions, we cannot use it if we run it through python.
+            This is described in more detail
+            :ref:`calling-c-functions`.
```

Comment:
```suggestion
            This is described in more detail at :ref:`calling-c-functions`.
```

Reply:
Look [here](https://github.com/cython/cython/issues/4187#issuecomment-861016107). This will be replaced

Reply:
[replaced](https://github.com/cython/cython/pull/4226/commits/b138847eb1098b57841abe05a6574ca3e827162f)